CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing to RKE #
+
+Thanks for taking the time to contribute to RKE!
+
+Of course, contributing code is more than welcome! To keep things simple, if you're fixing a small issue, you can simply submit a PR and we will pick it up. However, if you're planning to submit a bigger PR to implement a new feature or fix a relatively complex bug, please continue reading.
```

Comment:
> To keep things simple, if you're fixing a small issue, you can simply submit a PR and we will pick it up.
2 things:
1. I would still prefer to have an issue to track even "small issues". The only exceptions would be things like typos / misc changes. Anything that fixes an actual issue or implements a feature should an issue IMO.
2. I would remove "we will pick it up" as it establishes commitment to review any submitted PR.

Reply:
1. Yes, let's change that. The idea was to facilitate users contributing. Having to create an issue is another step which could be perceived as a burden. But as we don't expect RKE to become a "popular" open source project like k3s, we can remove that
2. I believe we should commit to review any submitted PR. Why not?